### ISSUE TITLE
Does not work with ipfs 0.7.0

### ISSUE THREAD

**Issue by xuv** [AuHau/ipfs-publish#29]
Got this error with the latest version of ipfs:
```
ERROR:publish.cli:Unsupported daemon version '0.7.0' (not in range: 0.4.22 ≤ … < 0.7.0)
```

**Comment by xuv** [comment]
This error seems to be coming from https://github.com/ipfs-shipyard/py-ipfs-http-client

**Comment by AuHau** [comment]
Should be solved thanks to updated dependencies. Closing this, but feel free to reopen if problem persists.

**Comment by xuv** [comment]
Hello @AuHau 

Sorry. But this isn't fixed. It still produces the same error with version 0.2.1.
The `ipfshttpclient` library that it uses is still the 0.6.1, which I believe has the error.
They've got a release candidate for version 0.7.0a1. But I haven't tried with that one

**Comment by AuHau** [comment]
I see. Hmm yeah, we will have to wait for stable release then. Thanks for letting me know!

**Comment by xuv** [comment]
Yes. It's been pushed. https://github.com/ipfs-shipyard/py-ipfs-http-client/pull/242

**Comment by AuHau** [comment]
Yes but that is not the final release though...

**Comment by xuv** [comment]
@AuHau I've tried with the pre-release. And now I get another error:
`ERROR:publish.cli:not pinned or pinned indirectly`

Haven't searched where it comes from. But you might want to take a look at it.

**Comment by xuv** [comment]
@AuHau Seems the final release is in. https://github.com/ipfs-shipyard/py-ipfs-http-client/releases/tag/0.7.0
Do you think you'd be interested in updating?

**Comment by xuv** [comment]
It seems that merging #46 would fix this. 
Tested by installing that branch and seems all is working as expected.

**Comment by AuHau** [comment]
Thanks for the heads-up! I will try to have a look on this soon.
